### PR TITLE
Dp 10107 refactor button sizing

### PR DESCRIPTION
**Pull request by clairesunstudio**
- Make current button "small" the default button size and add "large" and "small" variations.
 - Update the button size for location listing CTA to align with default size.

## Related Issue / Ticket

- Dp 10107
- #176 

## Steps to Test
Patternlab developer, please verify this change doesn't impact drupal location listing submit functionality --> https://github.com/massgov/mayflower/pull/177#discussion_r211316381

### CODE REVIEW

File: patternlab/styleguide/source/_patterns/02-molecules/location-filters.twig [hunk 1]
```diff
@@ -34,9 +34,15 @@
       </div>
     {% endif %}
     <div class="ma__location-filters__buttons">
-      <button class="ma__location-filters__submit" type="submit">
+      {% for button in locationFilters.buttons %}
```
Comment (clairesunstudio) [inline patternlab/styleguide/source/_patterns/02-molecules/location-filters.twig:37]: please verify this change doesn't have unexpected impact in drupal

Comment (ygannett) [reply patternlab/styleguide/source/_patterns/02-molecules/location-filters.twig:37]: In Drupal, currently only the submit button is implemented for location listing.  I don't see anything odd.  I don't think there is any impact in Drupal with this change.
Also, event listing page has no filter implemented.  

_qa_full_test_organization_locations___mass_gov_1

**Review by mrossi113 — CHANGES_REQUESTED** [review]
Seeing some impact to the "Tell Us What You Think" in the right corner within Drupal. 

screen shot 2018-08-22 at 11 59 34 am

**Comment by mrossi113** [comment]
Display in Production
prod

Display in my local with Mayflower change
local

**Review by ygannett — APPROVED** [review]
Look good to me.

**Comment by ygannett** [comment]
Please note that this change involves some Drupal side change.
PR:  [DP-10107 refactor button sizing #2684](https://github.com/massgov/mass/pull/2684)

**Comment by ygannett** [comment]
@mrossi113 As I checked some location listing pages in Drupal with this Mayflower version, I didn't see what you show.  Just making sure anything is missed during the review.

pattern_lab_-_pages-location-listing_and_massachusetts_registry_of_motor_vehicles_locations___mass_gov_and_1__npm_install____gulp__node_

**Comment by clairesunstudio** [comment]
@mrossi113 not sure why that feedback button looks like that... sounds like @ygannett didn't see that, do you mind pulling it and testing it again?

**Comment by mrossi113** [comment]
@clairesunstudio this is working now with the Drupal PR that @ygannett worked on https://github.com/massgov/mass/pull/2684. Sorry I did not see that PR when testing on Wednesday morning. I will approve this PR now, but the Drupal PR will need to be merged in when this Mayflower PR gets pulled into Drupal side.

**Review by mrossi113 — APPROVED** [review]

**Pull request merged**